apps/server/src/modules/news/mapper/news.mapper.spec.ts

```diff
@@ -62,33 +44,35 @@ const createNews = <T extends News>(
 		displayAt: date,
 		updatedAt: date,
 		createdAt: date,
-		school: schoolInfo.id,
-		creator: creatorInfo.id,
-		updater: creatorInfo.id,
+		school: school.id,
+		creator: creator.id,
+		updater: creator.id,
 		target: target.id,
 		...newsProps,
 	};
 	const resultNews = new NewsType(props);
-	resultNews.school = schoolInfo;
-	resultNews.creator = creatorInfo;
-	resultNews.updater = creatorInfo;
+	resultNews.school = school;
+	resultNews.creator = creator;
+	resultNews.updater = creator;
 	resultNews.target = target;
 	resultNews.targetModel = getTargetModel(resultNews);
 	return resultNews;
 };
 
 const getExpectedNewsResponse = (
-	schoolInfo: SchoolInfo,
-	creatorInfo: UserInfo,
+	school: School,
+	creator: User,
 	news: News,
 	newsProps: { title: string; content: string },
-	target: NewsTargetInfo
+	target: NewsTarget
 ): NewsResponse => {
-	const schoolInfoResponse: SchoolInfoResponse = new SchoolInfoResponse();
-	Object.assign(schoolInfoResponse, schoolInfo);
+	const schoolInfoResponse = new SchoolInfoResponse();
+	const schoolProps = (({ id, name }) => ({ id, name }))(school);
+	Object.assign(schoolInfoResponse, schoolProps);
 
 	const creatorResponse: UserInfoResponse = new UserInfoResponse();
-	Object.assign(creatorResponse, creatorInfo);
+	const creatorProps = (({ id, firstName, lastName }) => ({ id, firstName, lastName }))(creator);
```

Comment:
Why do you do this change?

Reply:
this is due to the refactoring from `creatorInfo: UserInfo` to `user: User`. the relevant properties are extracted from the user object.